### DISCUSSION TITLE
Broken Demo page

### DISCUSSION THREAD

**Discussion by Hullabalo** [JoshuaC215/agent-service-toolkit#47] [General]
Hello Joshua,
Thank you for your work on this template. Just for your information, the demo (https://agent-service-toolkit.streamlit.app/) is no longer working. A different error is returned depending on whether the streaming parameter is enabled or not.

**Comment by JoshuaC215** [comment]
Just needed a reboot, should be working now :) Thanks for reporting!

**Comment by Hullabalo** [comment]
Cool. I played around with it a bit, it works pretty well (when DuckDuckGo doesn't return a 202
error ... but I guess that's the price of success :)). I need to take a closer look at all this. Thanks again for your code. Have a great day!
